dell/precision/5530/default.nix

```
@@ -1,22 +1,47 @@
+{ lib, config, ... }:
+
 {
   imports = [
-    ../../../common/cpu/intel
     ../../../common/pc/laptop
     ../../../common/pc/laptop/ssd
+    ../../../common/gpu/nvidia/pascal
+    ../../../common/gpu/nvidia/prime.nix
+    ../../../common/cpu/intel/coffee-lake
   ];
 
-  boot.kernelParams = [
-    # fix lspci hanging with nouveau
-    # source https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1803179/comments/149
-    "acpi_rev_override=1"
-    "acpi_osi=Linux"
-    "nouveau.modeset=0"
-    "pcie_aspm=force"
-    "drm.vblankoffdelay=1"
-    "nouveau.runpm=0"
-    "mem_sleep_default=deep"
-    # fix flicker
-    # source https://wiki.archlinux.org/index.php/Intel_graphics#Screen_flickering
-    "i915.enable_psr=0"
-  ];
+  boot = {
+    kernelParams = lib.mkDefault [
```

For lists we should not use lib.mkDefault because it won't merge:

```suggestion
    kernelParams = [
```